src/Codata/Sized/Conat/Properties.agda

```diff
@@ -14,7 +14,8 @@ open import Codata.Sized.Thunk using (Thunk; Thunk^R; force)
 open import Codata.Sized.Conat
 open import Codata.Sized.Conat.Bisimilarity using (_⊢_≈_; refl; zero; suc)
 open import Function.Base using (_∋_)
-open import Relation.Nullary using (¬_; yes; no)
+open import Relation.Nullary.Decidable using (yes; no)
```

Comment:
except that `Relation.Nullary.Decidable` was already imported a couple of lines below... merge these two.